autoload/neomake/makers/ft/typescript.vim

```diff
@@ -8,9 +8,8 @@ function! neomake#makers#ft#typescript#tsc() abort
     " tsc should not be passed a single file.  Changing to the file's dir will
     " make it look upwards for a tsconfig.json file.
     return {
-        \ 'args': ['--noEmit'],
+        \ 'args': ['--project', getcwd() . '/tsconfig.json', '--noEmit'],
```

Comment:
Wouldn't just `./tsconfig.json` also work then?  (and allow to configure `neomake_typescript_tsc_cwd` if you want to)